README.md

```diff
@@ -22,22 +22,44 @@ Interne henvendelser kan sendes via Slack i kanalen #vtp-chatten
 
 # Hvordan komme igang - Oppsett og konfigurasjon
 
-* VTP trenger Keystore og Truststore liggende i mappe .modig på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
+* VTP trenger Keystore og Truststore liggende i mappe `.modig` på brukerens hjemme-mappe, eller konfigurert gjennom verdier angitt i avsnitt om sikkerhet.
 
-## Starte server fra IDE
+guide for å sette Keystore og Truststore: https://howtodoinjava.com/maven/how-to-install-maven-on-windows/
+
+* Intelliji må konfigureres med å sette classpath of module til server, og sette working directory til `$MODULE_WORKING_DIR$` i run/debug-konfiguration.
+
+## Teknologi, må være installert før du fortsetter
+* Node.js (https://nodejs.org/en/download/)
+* Yarn (https://classic.yarnpkg.com/en/docs/install/#windows-stable)
+* Java (https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
```

Comment:
For Java bør vi referere til OpenJDK: https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=hotspot

Reply:
Det er lurt, siden det er kan være misforståelse med det.